src/types.ts

```diff
@@ -57,16 +57,16 @@ export type NamedTensorMap = {
  * converted to a ConfigDict that has renamed fields (TS naming conventions)
  * and support for Enums.
  */
-export type JsonValue = boolean|number|string|null|JsonArray|JsonDict;
-export interface JsonDict {
-  [key: string]: JsonValue;
+export type PyJsonValue = boolean|number|string|null|PyJsonArray|PyJsonDict;
+export interface PyJsonDict {
```

Comment:
Same here. Please add a comment explaining what the "Py" part of the new interface name means.